src/Microsoft.PowerShell.Commands.Diagnostics/ExportCounterCommand.cs

```diff
@@ -162,7 +162,7 @@ protected override void BeginProcessing()
                 throw new PlatformNotSupportedException();
             }
 
-            // PowerShell Core requires at least Windows 7,
+            // PowerShell requires at least Windows 7,
```

Comment:
It is not right for 3.0 and etc. PowerShell 7 or PowerShell 6+ is better.
```suggestion
            // PowerShell 7 requires at least Windows 7,
```